apps/src/dance/ai/dance-ai-modal.module.scss

```diff
@@ -61,30 +65,69 @@
 
 .dialog {
   max-width: initial !important;
-  width: 800px !important;
-  height: 380px;
+  width: 824px !important;
+  height: 454px;
+  padding: 0 !important;
+  border: solid 1px $ai-block-border-color;
+  background-color: $ai-block-color !important;
+
+  .headerArea {
+    width: calc(100% - 40px);
+    height: 50px;
+    background-color: $ai-block-color;
+    color: white;
+    font-size: 16px;
+    display: flex;
+    align-items: center;
+    padding: 0 20px;
+    margin-top: 8px;
+
+    .botImage {
+      height: 40px;
+      margin-right: 20px;
+    }
+
+    .inputsContainer {
+      display: flex;
+      padding: 5px;
+      border-radius: 4px;
+      gap: 6px;
+      border: solid 2px $ai-block-inputs-border-color;
+
+      .emojiSlot {
+        width: 35px;
+        height: 35px;
+        margin-right: 6px;
+      }
+    }
+  }
 
   .innerArea {
-    width: 100%;
-    height: 100%;
+    width: calc(100% - 16px);
+    height: calc(100% - 50px - 16px - 8px);
```

Comment:
Similar to Ben's comment, I know the UI is very much subject to change but this feels like a lot of magic numbers - can we create constants or drop a comment for where these numbers come from?

Reply:
Good call.  Updated with both constants and comments.